source/main/config.js

```diff
@@ -104,7 +104,7 @@ export const ALLOWED_LOGS = [
   'Daedalus-versions.json',
   'State-snapshot.json',
 ];
-export const ALLOWED_NODE_LOGS = new RegExp(/(node.json-)(\d{14}$)/);
+export const ALLOWED_NODE_LOGS = new RegExp(/node.log-/);
```

Comment:
the new node will have the timestamp in between its log name "node" and the extension ".json". 
example: `node-20191113060708.json`
this made it easier for support to read the files, as Windows recognises the file extension.

Reply:
We should leave it as is for now as this is also how Daedalus logs are named...